packages/@sanity/desk-tool/src/panes/documentPane/changesPanel/changesPanel.css

```diff
@@ -56,8 +56,8 @@
 }
 
 .versionSelectContainer {
-  padding: var(--extra-small-padding) var(--small-padding);
-  padding-right: calc(var(--medium-padding) - var(--extra-small-padding) + 1px);
+  padding: 0 var(--small-padding);
+  /* padding-right: calc(var(--medium-padding) - var(--extra-small-padding) + 1px); */
```

Comment:
not sure to remove this or change this file in general, might be conflicts with other migrations? Same goes for the other files that are touched in this PR even though they don't directly involve the timeline